packages/core/providers/evaluation.ts

```diff
@@ -461,23 +458,44 @@ type PipelineRunTestCaseTuple = [PipelineRun, TestCase];
  */
 export const getTestRunners = async (
   pipeline: Pipeline<{ [key: string]: GentracePlugin<any, any> }>,
+  contextOrCaseFilter?:
+    | ResultContext
+    | ((
+        testCase: Omit<TestCase, "createdAt" | "updatedAt" | "archivedAt">,
+      ) => boolean),
+  caseFilterOrUndefined?: (
+    testCase: Omit<TestCase, "createdAt" | "updatedAt" | "archivedAt">,
+  ) => boolean,
 ): Promise<Array<PipelineRunTestCaseTuple>> => {
   if (!globalGentraceApi) {
     throw new Error("Gentrace API key not initialized. Call init() first.");
   }
 
+  const { context, caseFilter } = getContextTestCaseFilter(
```

Comment:
Isn't `context` only useful for test result submission? Why not exclude it from the function signature?

Reply:
yes, that was a mistaken inclusion.